drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/ChromeDriverBinaryHandler.java

```diff
@@ -127,6 +127,9 @@ public String getFileNameRegexToDownload(String version, Architecture architectu
             fileName.append("chromedriver_");
             if (PlatformUtils.isMac()) {
                 fileName.append("mac");
+                if (PlatformUtils.isMacAppleSilicon()) {
+                    fileName.append("_arm");
+                }
```

Comment:
On line [`141`](https://github.com/stringbean/arquillian-extension-drone/blob/c4105771715c151a8f70040e8677e16f81cd0f2c/drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/ChromeDriverBinaryHandler.java#L141-L143) there is also the following check:

```
if (PlatformUtils.isMacAppleSilicon()) {
	fileName.append("_m1");
}
```
Wouldn't it result in a wrong filename - with `_m1` added at the end?

I think the naming schema changed between the versions and if the version will be pinned to e.g. [`102.0.5005.61`](https://chromedriver.storage.googleapis.com/index.html?path=102.0.5005.61/) it will attempt to download the non-existing file.

The logic needs to handle such a case. It seems the last version using `_m1` suffix is [`106.0.5249.21`](https://chromedriver.storage.googleapis.com/index.html?path=106.0.5249.21/). We might want to consider a fallback with the suffix instead of assuming that from version X the new naming is used. Just an idea.